sklearn/datasets/_base.py

```diff
@@ -85,6 +85,11 @@ def clear_data_home(data_home=None):
     data_home : str or path-like, default=None
         The path to scikit-learn data directory. If `None`, the default path
         is `~/scikit_learn_data`.
+
+    Examples
+    ----------
```

Comment:
Though the PR has already been merged I think it's worth mentioning that the underline length is wrong, causing numpydoc to raise warning during doc build.

Reply:
Good catch. I will make a quick PR. Thanks.